prow/plugins/dco/dco.go

```diff
@@ -66,10 +67,27 @@ func init() {
 }
 
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
-	// The Config field is omitted because this plugin does not support
-	// per-repo config
+	configInfo := map[string]string{}
+	for _, orgRepo := range enabledRepos {
+		parts := strings.Split(orgRepo, "/")
+		var opts *plugins.Dco
+		switch len(parts) {
```

Comment:
@stevekuznetsov Do we have some helper that takes an orgRepo string and returns two strings, org and repo? I imagine this to be needed in many places